app/src/main/java/com/infomaniak/drive/ui/fileList/FileListFragment.kt

```diff
@@ -271,6 +272,8 @@ open class FileListFragment : MultiSelectFragment(MATOMO_CATEGORY), SwipeRefresh
         }
 
         setupBackActionHandler()
+
+        enqueueBulkDownloadWorker(requireContext(), folderId, userDrive ?: UserDrive())
```

Comment:
Check with `realm` if this forlder contains `marked offline files`

Reply:
Done